src/features/staging.ts

```diff
@@ -65,7 +66,7 @@ export const StagingMixin = <T extends Constructor<ModelViewerElementBase>>(
       super.updated(changedProperties);
 
       if (changedProperties.has('autoRotate')) {
-        (this as any)[$scene].pivot.rotation.set(0, 0, 0);
+        this[$scene].pivot.rotation.set(0, 0, 0);
```

Comment:
Should this be done using the new `setRotation` method? Seems like it would be better to let the `ModelScene` have full authority over the `pivot`'s rotation...

Reply:
Good call, I missed that one.